src/docs/forms/components/CheckBox/examples/02-CheckedByExternal.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Form, {
+    CheckBox,
+    TextInput
+} from 'doj-react-adminlte/Form';
+import ValueButton from "doj-react-adminlte/ValueButton";
+
+export default class CheckedByExternal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            checkbox1: null
+        };
+    }
+
+    handleChange = (name, value) => {
+        this.setState({
+            [name]: value
+        });
+    };
+
+    handleClick = () =>{
+      this.setState({checkbox1: true});
+    };
+
+
+    render() {
+        const {checkbox1} = this.state;
+        return (
+            <div>
+                <Form onChange={this.handleChange}>
+                    <div className="row">
+                        <div className="col-xs-3">
+                            <span className="text-bold">School Subjects</span>
+                            <CheckBox label="MATH"
+                                      name={checkbox1}
+                                      checked={checkbox1}/>
+                            <ValueButton className="clickToCheck"
```

Comment:
Use a regular `button` instead of a `ValueButton`. className to `btn btn-primary`.